frontend/packages/kubevirt-plugin/src/components/modals/modal/modal-footer.tsx

```diff
@@ -55,7 +59,9 @@ type ModalFooterProps = {
 
 export const ModalFooter: React.FC<ModalFooterProps> = ({
   id,
+  className = 'co-m-btn-bar modal-footer kubevirt-create-nic-modal__buttons',
```

Comment:
it would be better to use ```classNames('co-m-btn-bar modal-footer kubevirt-create-nic-modal__buttons', className)``` for the footer and leave className prop empty.

```
<footer className={classNames('co-m-btn-bar modal-footer kubevirt-create-nic-modal__buttons', className)}>
```